api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2426,14 +2428,29 @@ service AuthService {
   // DeleteTrustedCluster deletes an existing Trusted Cluster in a backend by name.
   rpc DeleteTrustedCluster(types.ResourceRequest) returns (google.protobuf.Empty);
 
-  // GetToken retrieves a token described by the given request.
+  // GetToken retrieves a v2 token described by the given request.
+  // Deprecated: use GetTokenV3.
+  // DELETE IN 13.0.
```

Comment:
Will this feature land in T11? If yes, usually we only keep backward compatibility for 1 version.